plugins/f1-scoreboard/manager.py

```
@@ -54,6 +54,16 @@ def __init__(self, plugin_id, config, display_manager,
         # Display duration
         self.display_duration = config.get("display_duration", 30)
 
+        # Resolve timezone: plugin config → global config → UTC.
+        # Inject into config so the renderer can convert UTC API times to local.
+        if not config.get("timezone"):
+            try:
+                global_tz = cache_manager.config_manager.get_timezone()
+            except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)